loader.tf

```diff
@@ -1,5 +1,6 @@
 resource "aws_lambda_function" "loader" {
   s3_bucket = local.lambda_buckets[var.aws_region]
+  # TODO: This needs to be updated
```

Comment:
Before we can commit this we'll need a new build of this zip file. The change from https://github.com/controlshift/aws-lambda-redshift-loader/pull/1 is needed to set DynamoDB table names.

Before this is commited, can ControlShift build a new zip and update the version here?

Reply:
The new zip file has been released. Thanks!